presto-main/src/test/java/com/facebook/presto/spiller/TestGenericPartitioningSpiller.java

```diff
@@ -57,7 +59,6 @@
     private static final int FOURTH_PARTITION_START = 20;
 
     private static final List<Type> TYPES = ImmutableList.of(BIGINT, VARCHAR, DOUBLE, BIGINT);
-    private static final PartitionFunction POSITIVE_AND_NEGATIVE_PARTITIONS_HASH_GENERATOR = new FourPartitionsPartitionFunction(0);
```

Comment:
Hardly related to the PR. I refactored the test class while adding more tests. This is all fixups.